src/apps/relay/mainrelay.c

```diff
@@ -1343,6 +1349,9 @@ static char Usage[] =
     "connections made to ports not\n"
     "						supporting HTTP. The default behaviour is to immediately "
     "close the connection.\n"
+    " --no-ratelimit-401\t\t\t\tDisable rate limiting for 401 Unauthorized responses\n"
```

Comment:
Please do not use inverse logic
This should not be default on anyway

`--ratelimit-401 Enable rate limiting....`

Reply:
FWIW the problem here is that not all coturn deployments are equal. This is a breaking change (i.e. if you enable this by default, a lot of applications straight up stop working - see my comment above, I've been fighting nextcloud-spreed for a better part of a quarter now and it still has a tendency to spontaneously fail when we least expect it ;_;)

I wanted to suggest adding a mention of this to the default example config... but I can't find one in the repo. oops?